packages/next/package.json

```diff
@@ -90,6 +91,7 @@
     "etag": "1.8.1",
     "events": "3.3.0",
     "find-cache-dir": "3.3.1",
+    "get-stream": "6.0.0",
```

Comment:
I think `raw-body` was supposed to be removed from dependencies, right? cc @timneutkens

Reply:
Yep! Done!